server/mdm/android/service/service.go

```diff
@@ -2,20 +2,23 @@ package service
 
 import (
 	"context"
-	"errors"
 	"fmt"
 	"net/http"
 	"strings"
 
+	"github.com/fleetdm/fleet/v4/server"
 	"github.com/fleetdm/fleet/v4/server/authz"
 	"github.com/fleetdm/fleet/v4/server/contexts/ctxerr"
 	"github.com/fleetdm/fleet/v4/server/fleet"
 	"github.com/fleetdm/fleet/v4/server/mdm/android"
 	"github.com/fleetdm/fleet/v4/server/mdm/android/service/proxy"
 	kitlog "github.com/go-kit/log"
-	"github.com/go-kit/log/level"
+	"google.golang.org/api/androidmanagement/v1"
 )
 
+// We use numbers for policy names for easier mapping/indexing with Fleet DB.
+const policyID = 1
```

Comment:
We'll always only have 1 policy? Again, I'd use a more precise name to avoid confusion.